css/theme-topic.css

```diff
@@ -164,11 +176,11 @@
 	font-size: 19px;
 	}
 
-.mobile-left {
+.experimental .mobile-left {
   float: left !important;
 }
 
-.contributors ul {
+.experimental .contributors ul {
 	margin-inline-start: 1.5em !important;
```

Comment:
Can we avoid the use of `!important` if the class name is this specific?

Reply:
Gone